packages/renderer/src/lib/feedback/feedbackForms/DevelopersFeedback.spec.ts

```diff
@@ -24,15 +24,19 @@ import { beforeAll, beforeEach, expect, test, vi } from 'vitest';
 
 import DevelopersFeedback from './DevelopersFeedback.svelte';
 
+const openExternalMock = vi.fn();
+const telemetryTrackMock = vi.fn();
+const sendFeedbackMock = vi.fn();
+
 beforeAll(() => {
   Object.defineProperty(window, 'openExternal', {
-    value: vi.fn(),
+    value: openExternalMock,
```

Comment:
issue: AFAIK we don't need to have this change

vi.mocked(window.openExternal) gives you access to the mock